src/jest/setup/request_animation_frame_polyfill.js

```diff
@@ -0,0 +1,18 @@
+// requestAnimationFrame isn't available in node so we need to polyfill it.
+// Borrowed from https://gist.github.com/paulirish/1579671.
+window.requestAnimationFrame = (() => {
+  let clock = Date.now();
+
+  return callback => {
+    const currentTime = Date.now();
+
+    if (currentTime - clock > 16) {
```

Comment:
We're only using this polyfill in test code, so I don't think we actually need to wait 16ms here. I suggest a simpler solution:

```js
// `jsdom` doesn't include `requestAnimationFrame`, so we include
// a simplified polyfill that's good enough for testing.
window.requestAnimationFrame = callback => {
  window.setTimeout(callback, 0);
}
```

Reply:
I tried testing your suggestion in https://github.com/elastic/kibana/pull/14183 and the tests failed. I think this is because the callback passed to `requestAnimationFrame` executed too quickly. Thoughts? Simulating it as closely as possible seems like a valid solution.

Reply:
Ah, yeah, just keep it as-is